README.md

```diff
@@ -446,15 +446,10 @@ Bug reports and pull requests are welcome on GitHub at https://github.com/github
 |@mellowfish|@horacio|@dukex|@dark-panda|@smashwilson|
```

Comment:
Might we add documentation for this feature to the README?

Reply:
👍Done. Take a read though and feel free to edit.